app/views/candidate_interface/submitted_application_form/start_carry_over_between_cycles.html.erb

```diff
@@ -5,16 +5,36 @@
 <div class="govuk-grid-row">
 
   <div class="govuk-grid-column-two-thirds">
-    <p class="govuk-body">Your last application has been saved, so you do not need to start again.</p>
-    <p class="govuk-body">You can make changes to your application whenever you like.</p>
-    <p class="govuk-body">You will be able to choose courses from <%= EndOfCycleTimetable.find_reopens.to_s(:govuk_date) %> and submit from <%= EndOfCycleTimetable.apply_reopens.to_s(:govuk_date) %>. </p>
-    <p class="govuk-body">You'll have 3 course choices.</p>
 
-    <%= button_to 'Start now', candidate_interface_carry_over_path, class: 'govuk-button govuk-button govuk-!-margin-bottom-5' %>
+    <% if EndOfCycleTimetable.find_down? %>
```

Comment:
@EmmaFrith @adamsilver - in the Trello ticket the requirement is to add this copy if before Oct 6th, but should this actually be the apply 2 deadline (i.e Sep 18th)?

Reply:
Fixed! 

All date formats now fixed too (they now adopt the YYYY to YYYY format)